project/duties.py

```diff
@@ -299,7 +323,10 @@ def release(ctx, version):
         ctx.run("git push --tags", title="Pushing tags", pty=False)
         ctx.run("poetry build", title="Building dist/wheel", pty=PTY)
         ctx.run("poetry publish", title="Publishing version", pty=PTY)
-        ctx.run("mkdocs gh-deploy", title="Deploying documentation", pty=PTY)
+        if GITHUB_CI:
+            ctx.run("mkdocs gh-deploy", title="Deploying documentation", pty=PTY)
+        elif GITLAB_CI:
+            ctx.run("mkdocs build -s --site-dir public", title="Deploying documentation")
```

Comment:
Again, this should be templated instead of decided at runtime.
Also, I think this can be simplified by directly calling the `docs_deploy` action:

```python
docs_deploy.run()  # use its own context
# or docs_deploy(ctx)  # use the current context
```

(possible since duty v0.6.0)

Reply:
removed and put this which with above tweak in mkdocs.yml will build to proper path without specifying it in the command.

ctx.run("mkdocs build", title="Deploying documentation")